### PR TITLE
Propose some references about ci/cd solutions compatibles with gitea

### PR DESCRIPTION
**Pull request by camlafit**
* No note about theirs integrations level, it's only a starting point.

### CODE REVIEW

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    identifier: "ci-cd"
+---
+
+# Gitea and CI/CD
+
+NOTE: These tools are not endorsed by Gitea. They are listed here for convenience only.
+
+## Listing
+
+Some CI/CD solutions are compatibles with Gitea. As following list can't be complete, the purpose is to give a starting point to integrate a CI/CD process in your gitea.
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
CI/CD solutions that have integration with Gitea. Following list is not complete,
the purpose is to give a starting point to integrate a CI/CD process with Your Gitea instance.
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+ - Drone : https://drone.io
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Drone](https://drone.io/)
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ - Jenkins : https://plugins.jenkins.io/gitea
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Jenkins](https://jenkins.io/) with [Gitea plugin](https://plugins.jenkins.io/gitea)
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ - Agolia :  https://agola.io
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Agolia](https://agola.io/)
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ - Buildkite : https://buildkite.com and https://github.com/techknowlogick/gitea-buildkite-connector
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Buildkite](https://buildkite.com/) with [Gitea connector](https://github.com/techknowlogick/gitea-buildkite-connector)
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+
+Anothers CI/CD could be work with gitea as :
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
Others CI/CD solutions that partially can be integrated with Gitea:
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ - concourse-ci : https://www.concourse-ci.org and https://discuss.concourse-ci.org/t/concourse-ci-and-gitea-oauth/1475
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Concourse](https://concourse-ci.org/), see more info at [Concourse community forum](https://discuss.concourse-ci.org/t/concourse-ci-and-gitea-oauth/1475)
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Comment by camlafit** [comment]
Hello

I've updated texts as suggested 

Thanks

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+NOTE: These tools are not endorsed by Gitea. They are listed here for convenience only.
+
+## Listing
+
+CI/CD solutions that have integration with Gitea. Following list is not complete,
+the purpose is to give a starting point to integrate a CI/CD process with Your Gitea instance.
+
+ - [Drone](https://drone.io)
+ - [Jenkins](https://plugins.jenkins.io/gitea)
+ - [Agolia](https://agola.io)
```
[the remark this answers is not in the document]
Comment (camlafit) [reply docs/content/doc/advanced/ci-cd.en-us.md]: Yes ... I was thinking about algolia :/ 
Thanks

**Comment by camlafit** [comment]
@techknowlogick but when i do my search to have an idea about compatible CI ecosystem, I haven't found this page. Act as an hidden page, is not intuitive to check this page.

As CI tools looks as a source management continuity, I think could be better to propose a specific page.

**Comment by lafriks** [comment]
I agree about moving CI/CD to separate page. This PR should replace CI section with link to new page from 3rd party tools page

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+CI/CD solutions that have integration with Gitea. Following list is not complete,
+the purpose is to give a starting point to integrate a CI/CD process with Your Gitea instance.
+
+ - [Drone](https://drone.io)
+ - [Jenkins](https://plugins.jenkins.io/gitea)
+ - [Agola](https://agola.io)
+ - [Buildkite](https://buildkite.com) with [Gitea connector](https://github.com/techknowlogick/gitea-buildkite-connector)
+
+
```
Comment (appleboy) [inline docs/content/doc/advanced/ci-cd.en-us.md]: remove the redundant empty line.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: docs/content/doc/advanced/ci-cd.en-us.md (hunk 9 above)
Comment (camlafit) [reply docs/content/doc/advanced/ci-cd.en-us.md]: Patch done thanks

**Comment by camlafit** [comment]
> Please disregard my last comment. @lafriks has suggested a better way.

I've update third party page with a replacement link. Hope it's correct.

**Review by jolheiser — CHANGES_REQUESTED** [review]
Agree having a CI/CD page would be good for clarity. Just a few nits, thanks for the PR!

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+# Gitea and CI/CD
+
+NOTE: These tools are not endorsed by Gitea. They are listed here for convenience only.
+
+## Listing
+
+CI/CD solutions that have integration with Gitea. Following list is not complete,
+the purpose is to give a starting point to integrate a CI/CD process with Your Gitea instance.
+
+ - [Drone](https://drone.io)
```
[the remark this answers is not in the document]
Comment (camlafit) [reply docs/content/doc/advanced/ci-cd.en-us.md]: To continue with others CI, I've maintained CI link and add specific gitea link

**Comment by camlafit** [comment]
Suggestions applied thanks

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+  sidebar:
+    parent: "advanced"
+    name: "CI/CD Usage"
+    weight: 40
+    identifier: "ci-cd"
+---
+
+# Gitea and CI/CD
+
+NOTE: These tools are not endorsed by Gitea. They are listed here for convenience only.
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
**NOTE:** These tools are not endorsed by Gitea. They are listed here for convenience only.
```

File: docs/content/doc/advanced/ci-cd.en-us.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+CI/CD solutions that have integration with Gitea. Following list is not complete,
+the purpose is to give a starting point to integrate a CI/CD process with your Gitea instance.
+
+ - [Drone](https://drone.io) with [Gitea documentation](https://docs.drone.io/installation/gitea/)
+ - [Jenkins](https://jenkins.io/) with [Gitea plugin](https://plugins.jenkins.io/gitea)
+ - [Agola](https://agola.io)
+ - [Buildkite](https://buildkite.com) with [Gitea connector](https://github.com/techknowlogick/gitea-buildkite-connector)
+
+Others CI/CD solutions that partially can be integrated with Gitea:
+ - [Concourse](https://www.concourse-ci.org), see more informations at [Concourse community forum](https://discuss.concourse-ci.org/t/concourse-ci-and-gitea-oauth/1475)
```
Comment (lafriks) [inline docs/content/doc/advanced/ci-cd.en-us.md]:
```suggestion
 - [Concourse](https://www.concourse-ci.org), see more information at [Concourse community forum](https://discuss.concourse-ci.org/t/concourse-ci-and-gitea-oauth/1475)
```

**Comment by camlafit** [comment]
Patches applied

**Review by techknowlogick — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Pull request merged**